docs/shader-modules/project.md

```diff
@@ -48,7 +48,12 @@ The GLSL uniforms of the `project` module are quite central to shader code and a
 | project_uViewportSize | vec2 | size of viewport in pixels |
 | project_uDevicePixelRatio | float | device pixel ratio of current viewport (value depends on `useDevicePixels` prop) |
 | project_uFocalDistance | float | distance where "pixel sizes" are display in 1:1 ratio (modulo `devicePixelRatio`) |
-| project_uCameraPosition | float | position of camera in world space |
+| project_uCameraPosition | vec3 | position of camera in world space |
```

Comment:
Good discussion point... I left these uniforms out initially because I felt they were less likely to be needed by apps and might be considered implementation details. Since they would normally always be handled by the API functions. 

Maybe make a note under each Uniforms section along these lines?
> Uniform names are not considered part of the official API of shader modules and can potentially change between minor releases, but are documented for applications that need this level of access. Use the module's public GLSL functions instead of directly accessing uniforms when possible.

Reply:
We should probably also document the uniforms under the functions for this reason. First JS API, then GLSL functions, then GLSL uniforms...